python/ray/serve/tests/test_cli.py

```diff
@@ -700,5 +700,36 @@ def test_deploy_from_import_path(ray_start_stop):
     )
 
 
+@pytest.mark.skipif(sys.platform == "win32", reason="File path incorrect on Windows.")
+def test_deploy_with_access_to_current_directory():
+    """Test serve deploy with using current directory is able to deploy the app.
+
+    We had issue where dashboard client no longer has the current working added to
+    the sys.path and unable to deploy Serve application in the current directory. This
+    test will ensure the files in the current directory can be accessed and deployed.
+    """
+    # Stop any existing ray session if any.
+    subprocess.check_output(["ray", "stop"])
```

Comment:
this shouldn't be needed (unless other tests are not properly using fixtures)

speaking of which, this test needs to use a fixture to stop ray. otherwise if it fails it will cascade onto other tests.